kolibri/plugins/facility/assets/src/views/FacilityConfigPage/ConfigPageNotifications.vue

```diff
@@ -1,58 +1,16 @@
-<template>
-
-  <div>
-    <UiAlert
-      v-if="notification === notificationTypes.PAGELOAD_FAILURE"
-      type="error"
-      @dismiss="dismiss()"
-    >
-      {{ $tr('pageloadFailure') }}
-    </UiAlert>
-
-    <UiAlert
-      v-if="notification === notificationTypes.SAVE_SUCCESS"
-      type="success"
-      @dismiss="dismiss()"
-    >
-      {{ $tr('saveSuccess') }}
-    </UiAlert>
-
-    <UiAlert
-      v-if="notification === notificationTypes.SAVE_FAILURE"
-      type="error"
-      @dismiss="dismiss()"
-    >
-      {{ $tr('saveFailure') }}
-    </UiAlert>
-  </div>
-
-</template>
-
-
 <script>
 
-  import UiAlert from 'kolibri-design-system/lib/keen/UiAlert';
-  import { notificationTypes } from '../../constants';
+  // TODO: This component is kept for $trs strings, delete on 0.15 version
```

Comment:
Nice, this is exactly what we've had to do a number of times because it's hard to move translated messages around 👍